python_modules/libraries/dagster-databricks/dagster_databricks/components/databricks_asset_bundle/component.py

```diff
@@ -84,6 +111,24 @@ class DatabricksAssetBundleComponent(Component, Resolvable):
             ],
         ),
     ]
+    cluster_config: Annotated[
+        DatabricksClusterConfig,
+        Resolver(
+            resolve_cluster_config,
+            model_field_type=Optional[DatabricksClusterConfigArgs.model()],
+            description=(
+                "A mapping defining a databricks_asset_bundle.configs.ClusterConfig. Optional."
+            ),
+            examples=[
+                {
+                    "spark_version": "test_spark_version",
+                    "node_type_id": "node_type_id",
+                    "num_workers": 1,
+                },
+                None,
+            ],
+        ),
+    ]
```

Comment:
make sure to give this a default value of `DatabricksClusterConfig()` so people using the python API don't have to specify this